README.md

```diff
@@ -28,6 +28,10 @@ Make sure:
 - You have few knowledge about Html, Css, flexbox, media queries, grid, Github and Linters
 - You have installed Live Server to view the project in the browser.
 
+### VIEW DEMO
```

Comment:
- [ ] **Required:** You have done excellently, however you need to fix the title of your website. https://github.com/KABIKA681/portfolio-pascal-kabika/blob/4754999d7093226f0f9a4b9b1626ee328de7fc3d/index.html#L16
At the moment it is the default `Document`. Let's update it to Kabika's Portfolio, Kabika Pascal Portfolio, etc. for example.